specs/array_of_g_ids.go

```diff
@@ -0,0 +1,26 @@
+package specs
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	strfmt "github.com/go-openapi/strfmt"
+
+	"github.com/go-openapi/errors"
+)
+
+/*ArrayOfGIds array of g ids
+
+swagger:model ArrayOfGIDs
```

Comment:
@stevvooe please refer to comments above about using swagger.. (this is just a test commit to discuss if this is the way we would go with this generation)